resource_test.go

```diff
@@ -80,6 +80,40 @@ func TestDefaultMapResource(t *testing.T) {
 				},
 			},
 		},
+		{
+			input: &resource.Resource{
+				Type: resourcekeys.HostType,
+				Labels: map[string]string{
+					stackdriverProjectID:           "proj1",
+					resourcekeys.K8SKeyClusterName: "cluster1",
+					resourcekeys.CloudKeyZone:      "zone1",
```

Comment:
Please also add a host name label in this test.

Reply:
Added